### ISSUE TITLE
Arma Reforger support missing

### ISSUE THREAD

**Issue by noaim** [Austinb/GameQ#603]

**Comment by bumbummen99** [comment]
Can you try with generic `source` protocol? As ArmA3 protocol does extend source it did always work for me tought it lacks extended information of the query.

**Comment by bumbummen99** [comment]
Looks like Reforger does have `A2S` but it is disabled by default. According to discord you can enable and bind it to a port with the following parameter [`-a2sPort <PORT>`](https://community.bistudio.com/wiki/Arma_Reforger:Startup_Parameters#a2sPort)

**Comment by noaim** [comment]
i have set -a2sIpAddress and -a2sPort and it doesnt work.

**Comment by bumbummen99** [comment]
> i have set -a2sIpAddress and -a2sPort and it doesnt work.

Yeah according to the ArmA discord we will have to wait a while until stuff becomes more sorted out. That A2S is not enabled by default yet does tell me that it is probably not even stable or working at all.

**Comment by noaim** [comment]
"a2sQueryEnabled": true,
    "steamQueryPort": 17777

ive added this to the server config json file, the a2sport aint needed. if i use other server viewer it works, but shows strange server name and strange map name

**Comment by Austinb** [comment]
Is there any update on this issue.  Is Arma still working properly or is it broken?
